portfolio/src/main/webapp/script.js

```diff
@@ -29,4 +30,42 @@ function respondToClick(clickEvent) {
   } else {
     content.style.display = "block";
   }
-}
\ No newline at end of file
+}
+
+let submitButton = document.getElementById("post");
+submitButton.addEventListener("click", postComment);
```

Comment:
Can this not go in the html as an onClick?

Reply:
I think it is now preferable to avoid referencing js directly in html files, and instead using event listeners to handle changes. These two links seem to corroborate that, let me know what you think
[Unobtrusive JS](https://en.wikipedia.org/wiki/Unobtrusive_JavaScript)
[S.O. answer on why HTML onClick is not good practice](https://stackoverflow.com/questions/5871640/why-is-using-onclick-in-html-a-bad-practice)

Reply:
Thanks, didn't know about this!